frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -486,9 +486,14 @@ window.countlyManagementView = countlyView.extend({
 
     beforeRender: function() {
         var self = this;
-        return $.when(T.render(this.templatePath, function(src) {
-            self.template = src;
-        }));
+        if (this.templatePath && this.templatePath !== "") {
```

Comment:
If templatePath is truthy, it can't be empty string, so this would behave identical:

```js
if (this.templatePath) {
```

Reply:
better being thorough than sorry? :D

Reply:
At least someone does actually review :D

Reply:
@ar2rsawseen The `!== ""` test is a no-op, so it's not really "thorough" IMHO :)

Reply:
Operation wise, sure
but it is more readable, as I can understand type and requirements by looking just on that case

Reply:
Honestly, that thing makes it way less readable